aircraft/global5000/scripts/trim-cruise.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
+<?xml-stylesheet type="text/xsl" href="http://jsbsim.sf.net/JSBSimScript.xsl"?>
+<runscript xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="http://jsbsim.sf.net/JSBSimScript.xsd"
+    name="Trim in Cruise configuration">
+
+  <description>
+    Trimmed at altitude/airspeed in Cruise configuration, 
+    run for 1 second
+
+      Author:
+    Bill Galbraith
+    Holy Cows, Inc.
+    billg@holycows.net
+    December 2010
+    
+      Modified by:
+    Guilherme Lima da Silca
+    ATS aerothermalsolutions.co 
+    aerospace@ats4i.com 
+    May 2021
+    
+  </description>          
+
+  <use aircraft="global5000" initialize="scripts/airborne"/>
```

Comment:
```suggestion
  <use aircraft="global5000" initialize="airborne"/>
```
Initial conditions files should be located in the folder `aircraft/global5000` to keep consistence with other aircraft.
Please update the line above as suggested and move the file `airborne.xml` from `scripts/` to `aircraft/global5000`
```bash
> git mv scripts/airborne.xml aircraft/global5000/airborne.xml
```